### PR TITLE
Add a simple method to switch headless mode.

### PR DESCRIPTION
**Pull request by Keoghan**
Just a thought as to how we could make switching in/out of headless mode a little easier. Might be more worthy of a Dusk PR than doing it here.

### CODE REVIEW

File: src/Options.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+namespace Orchestra\Testbench\Dusk;
+
+use Facebook\WebDriver\Chrome\ChromeOptions;
+
+class Options
+{
+    static $headless = true;
+
+    public static function headless()
```
Comment (crynobone) [inline src/Options.php]: should be `withUI()`

File: src/Options.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    {
+        static::$headless = true;
+    }
+
+    public static function notHeadless()
```
Comment (crynobone) [inline src/Options.php]: should be`withoutUI()`

File: src/Options.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    {
+        static::$headless = false;
+    }
+
+    public static function isHeadless()
```
Comment (crynobone) [inline src/Options.php]: should be `hasUI()`

**Comment by Keoghan** [comment]
I agree with all of those @crynobone, will take a look at the --disable-gpu option as I'm not sure on that at the moment.

**Comment by crynobone** [comment]
If we have this, we could suggest developer to use `bootstrap` file for phpunit to make this configuration.

**Comment by Keoghan** [comment]
Based on my reading around --disable-gpu I agree it appears to only be required with --headless, so I have altered as required.

**Pull request merged**